health/processes.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+import psutil
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_all_processes():
+    """
+    Queries for all processes on the current system and handles any exceptions that may occur.
+
+    NOTE: On systems running more than Quay, this may include processes unrelated to Quay.
+    """
+    try:
+        return [p for p in psutil.process_iter()]
+    except Exception as e:
+        logger.exception("Unable to list all processes")
+    return []  # Don't allow this to break the application
```

Comment:
This whole file is a lot of boiler plate for 3 extremely simple list comprehensions.

I'm not sure why you wouldn't want to still raise this exception. Not being able to list processes on the system is a pretty bad failure and I'd rather that be explicit than have a function lie to me and tell me that there are 0 processes running or that there are 0 zombies when they're actually could be. I'd catch the exception in the worker and _not_ push a value as a result, rather than pushing a value of 0 to the gauge that's just plain wrong.